molo/profiles/templates/profiles/change_password.html

```diff
@@ -21,22 +21,22 @@
     </ul>
   {% endif %}
 
-   <div>
-      <label class="label-wide">{% trans "Old Password:" %}</label>
+   <div class="profile__fieldset">
```

Comment:
There's a `fieldset` tag, maybe we should consider using it instead? What are you thoughts on this?